helpers.lua

```diff
@@ -586,7 +586,7 @@ function get_data_for(view)
     data['flash_fail'] = {flashs["fail"]}
     data['flash_win']  = {flashs["win"] }
     data['flash_info'] = {flashs["info"]}
-    data['theme_name'] = ""..conf["theme"]
```

Comment:
Sorry, that was a debugging measure I forgot to change back. (I don't know lua at all so I had to try some stuff)

Reply:
Also it didn't work for me *at all* without the stuff that I consider to be a typecast to String.

Reply:
Uh that's weird o.O It seems to be working on my side ?

Reply:
Okay, Ignore my comment then :grin: